go-apps/meep-rnis/api/swagger.yaml

```diff
@@ -3633,28 +3633,6 @@ components:
       - 30
       - 31
```

Comment:
_Details_ section should be replaced with _Notes_ section from MEC Sandbox API files.

Reply:
as per above